revolv/templates/base/partials/dashboard_sidebar.html

```diff
@@ -0,0 +1,72 @@
+{% comment %}
+    Dashboard Sidebar
+
+    This is a partial for the sidebar of the RE-volv dashboard design (mockup can be seen at
+    https://cloud.githubusercontent.com/assets/7706852/6608266/b4c8fdd8-c802-11e4-9610-939a5f9f5dd9.png).
+    This partial accepts the following template variables:
+
+    :user: the currently logged in user. The user will always be logged in, since if they're
+            seeing this page then they have bypassed the functionality which redirects to the
+            homepage for unauthenticated users.
+
+    :project_dict: a dictionary with keys as project group names, such as "Completed Projects", "Projects",
+            etc, and values as lists of project model objects in the group. The project group names
+
+    The idea for the sidebar is that there is an area at the top where the user can switch between
+    different user types, if applicable, can see the projects which are relevant to their role, and can also
+    see management pages (for example admin accounting page) below. The point of this template is to
+    conditionally render all of these partials and things based on what type of user the current user is.
+{% endcomment %}
+{% spaceless %}
+<div class="role-select">
+    {% if role == "administrator" and user.revolvuserprofile.is_administrator %}
```

Comment:
Yeah, because the admin might not be viewing it as the sane role that s/he actually is